torch/multiprocessing/__init__.py

```diff
@@ -22,11 +22,19 @@
            'get_all_sharing_strategies']
 
 
-from multiprocessing import *  # noqa: F403
-
-
-__all__ += multiprocessing.__all__  # noqa: PLE0605 type: ignore[attr-defined]
-
+# Compat from older version of python to make sure we get the objects
+# compatible with the default context.
+import multiprocessing as mp
```

Comment:
Hmm i have an idea, though to be fair I have 20% confidence it may change things. Could you try not naming this mp and just import what you need here?

Also, we're trying to not care about older pythons (we support 3.7+) or might even be 3.8 now and we'd prefer not having .pyi files. So if this code were to be landed, let's merge the types in the .pyi back into the .py.